src/utils/getName.ts

```diff
@@ -2,7 +2,7 @@ import * as ts from "typescript";
 
 export default function getName(name): any {
   if (name.indexOf("-") !== 0) {
-    return ts.createLiteral(name);
+    return ts.factory.createStringLiteral(name);
```

Comment:
is this intentional change? 

previously if name had hyphen it returned literal otherwise it returned identifier now it always returns string literal? If its intentional then this function contents could be replaced with ts.factory.createStringLiteral(name)